controllers/account-setting/my-team/remove-access-from-myteam.controller.js

```diff
@@ -10,6 +10,13 @@ const removeAccessFromMyTeam = async (req, res, next) => {
 		const { disconnectUserId } = req.body;
 		const currentUserDetails = await UserModel.findById(user.id);
 
+		if (user.role !== 'CRM_ADMIN' && user.role !== 'CRM_SUPERVISOR') {
```

Comment:
This is redundant and bad practise. use a middleware. Already implemented with routes in notifications or auth